src/app/components/Item/ItemTableRow.jsx

```diff
@@ -34,9 +34,8 @@ const ItemTableRow = ({ item, bibId, getRecord }) => {
     <tr className={item.availability}>
       <td>{item.location}</td>
       <td>{itemDisplay}</td>
-      <td>{item.status.prefLabel}</td>
```

Comment:
The problem with this is that they want the status message OR the request button if the status message is "available". The logic is not in place for this so let's just show everything.